pulse-rs/src/lib.rs

```diff
@@ -604,6 +605,13 @@ impl ChannelMapExt for ChannelMap {
         }
         cm
     }
+    fn init_auto(ch: u32, def: ffi::pa_channel_map_def_t) -> bool {
```

Comment:
There's no way to get the initialized ChannelMap from this, so although it fits the use in PulseStream::stream_init below, it doesn't really match the PA API or the behaviour of ChannelMapExt::init.

I think it might be better if the signature was `fn init_auto(ch: u32, def: ffi::pa_channel_map_def_t) -> Option<ChannelMap>;`; that allows it to be used like ChannelMapExt::init and you can change the use in PulseStream::stream_init to use is_none().

Reply:
Good point, thanks